src/screens/HomeScreen/component.js

```diff
@@ -0,0 +1,128 @@
+// @flow
+import React, { PureComponent } from "react";
+import {
+  StyleSheet,
+  SafeAreaView,
+  ScrollView,
+  View,
+  TouchableOpacity
+} from "react-native";
+import type { NavigationScreenProp, NavigationState } from "react-navigation";
+import Text from "../../components/Text";
+import type { Event, Asset } from "../../data/event";
+import EventTile from "../../components/EventTile";
+import {
+  cardBgColor,
+  imageBgColor,
+  titleTextColor
+} from "../../constants/colors";
+import { FEATURED_EVENT_LIST, EVENT_DETAILS } from "../../constants/routes";
+
+const locale = "en-GB";
+
+type Props = {
+  navigation: NavigationScreenProp<NavigationState>,
+  featuredEventsTitle: string,
+  featuredEvents: Event[],
+  loading: boolean,
+  getAssetById: string => Asset
+};
+
+class HomeScreen extends PureComponent<Props> {
+  static navigationOptions = {
+    header: null
+  };
+
+  eventDetails = (eventId: string) => {
+    this.props.navigation.navigate(EVENT_DETAILS, { eventId });
+  };
+
+  eventList = () => {
+    this.props.navigation.navigate(FEATURED_EVENT_LIST, {
+      title: this.props.featuredEventsTitle
+    });
+  };
+
+  render() {
+    // Show only even number of events (2, 4 or 6).
+    // Never show more than 6 events.
+    const eventsCount = Math.min(
+      6,
+      this.props.featuredEvents.length - this.props.featuredEvents.length % 2
```

Comment:
Might be more readable as `Math.floor(this.props.featuredEvents.length / 2)`

Reply:
Agree. I'm going to change this.